app/services/PaypalService.scala

```diff
@@ -137,7 +138,7 @@ class PaypalService(config: PaypalApiConfig, contributionData: ContributionData)
         created = created,
         email = payerInfo.getEmail,
         ophanId = ophanId,
-        abTests = chosenVariants.asJson,
+        abTests = Json.toJson(variant),
```

Comment:
I know it's going to sound crazy, but I think if we want the DB to be backward compatible, we need that value to be wrapped into an array. I think something like that should be possible

```
Json.Array(Json.toJson(variant))
```

or

```
Json.toJson(Seq(variant))
```